tools/status_codes/.gitignore

```diff
@@ -0,0 +1 @@
+Cargo.lock
```

Comment:
I think we are supposed to commit Cargo.lock files.

Reply:
> 1. Can you add a line to the capsules readme with the low_level_debug capsule, and a short readme in the capsule folder to help curious capsule-folder-navigators?

Done

> 2. I think having status codes and error codes will be confusing, particularly with the tool to print status codes. I don't really know what to do about that, however. Renaming status codes to something with "app" in it would help, but then I suspect people would expect that to be `main()`'s return value.

Perhaps I should replace "status code" with "alert code"?

Reply:
> I think we are supposed to commit Cargo.lock files.

Pros of commiting Cargo.lock files:

  1. Easy way to get reproducible builds.

Cons of committing Cargo.lock files:

  1. Makes offline builds a nightmare. Cargo gets really weird/buggy/picky when a Cargo.toml is present. e.g. if this binary picks up a dependency in common with elf2tab then the tock-on-titan project would need to fork one or the other to build both.
  2. Checking in build artifacts, with the associated code review spam (non-reviewable changes in Cargo.lock when you update cargo, for example).

The general guidance for cargo is to check in Cargo.lock files for binaries, but that guidance is not explained well. Would you prefer I change it or keep it as-is?

Reply:
`.lock` files are not my friend (for some reason I seem to be the only one who causes them to regenerate and thus show as changed files in git; I'm always causing them to be modified for completely unrelated changes), so I don't really know.

Reply:
I think alert code would work well.

Reply:
I've committed Cargo.toml. For some reason GitHub hides its contents, but that's out of the scope of this PR.